lib/tasks/tol_tubes_report_view.rake

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+namespace :tol_tubes_report_view do
```

Comment:
Did you look at whether we could do rspec tests on this? We have a few rake task specs to base it on but I understand this is slightly different because of the raw sql.

Reply:
I investigated adding tests and hit a few road blocks with testing the actual view data.  As we discussed, it's looking unlikely that we can test the data without some test packages being modified which is a bit of an undertaking for a view that's supposed to be quickly implemented.  When the test creates or destroys the view, the transaction that would otherwise be applied to the test data is overridden and commited to the database permanently across tests and test runs.  One solution is DatabaseCleaner but I think that's too far for this issue.